docs/rfcs/XeGPU.md

```diff
@@ -412,10 +474,22 @@ Below example shows that each WI loads 4 fp32 data element with the chunk_size.
           tensor_desc<16x4xfp32, #tdesc_attr, #sg_map_t>, vector<1xi1> -> vector<4x1xfp32>
 ```
 
+The load with chunk_size pack the low-precision data to 32-bit data using wi_data = [1, 2].
+```mlir
+  #sg_map_t = xegpu.sg_map<wi_layout = [16, 1], wi_data = [1, 2]>
+  #scatter_attr = !xegpu.tdesc_attr< memory_space=slm, scattered=true>
+  %scatter_tdesc_chunk = xegpu.create_tdesc, %src_addr, %offsets
+		{chunk_size=4} :
+		uint64, vector<16xindex> into tensor_desc<16x8xbf16, #scatter_attr, #sg_map_t>
+
+  %result = xegpu.load_gather %scatter_tdesc_chunk, %mask {L1 = cached, L2 = uncached, transpose=[1,0]} :
```

Comment:
btw I think in the upstream XeGPU def we have operation just named "load" instead of "load_gather"

Reply:
OK. I will change the RFC to be consistent. 

https://mlir.llvm.org/docs/Dialects/XeGPU/#xegpuload-xegpuloadgatherop
xegpu.load (xegpu::LoadGatherOp)

Reply:
In a separate PR.